ethercat_iface_name: enp0s31f6 #default: eth0

min_acceptable_rt_loop_frequency: 490.0

halt_on_slow_rt_loop: false

publish_every_rt_jitter: false

number_of_controller_boards: 1

analog_inputs:
  - name: analog_input_1
    controller_board: 0 #0....{number_of_controller_boards - 1}
    controller_channel: 0 #0.....31
    publish_muscle: false
  - name: analog_input_2
    controller_board: 0
    controller_channel: 1
    publish_muscle: false

muscle_list:
  - name: muscle_1
  - name: muscle_2
  - name: muscle_3
  - name: muscle_4
  - name: muscle_5
  - name: muscle_6
  - name: muscle_7
  - name: muscle_8
  - name: muscle_9
  - name: muscle_10
  - name: muscle_11
  - name: muscle_12
  - name: muscle_13
  - name: muscle_14
  - name: muscle_15
  - name: muscle_16
  - name: muscle_17
  - name: muscle_18
  - name: muscle_19
  - name: muscle_20
  - name: muscle_21
  - name: muscle_22
  - name: muscle_23
  - name: muscle_24
  - name: muscle_25
  - name: muscle_26
  - name: muscle_27
  - name: muscle_28
  - name: muscle_29
  - name: muscle_30
  - name: muscle_31
  - name: muscle_32
